src/api/app/views/webui2/webui/project/_pulse_list.html.haml

```diff
@@ -0,0 +1,91 @@
+.row
+  .col
+    %p
+      During this period
+      = render partial: 'pulse_list_commits'
+    %p
+      = render partial: 'pulse_list_builds'
+      = render partial: 'pulse_list_new'
+      = render partial: 'pulse_list_changes'
+    %p
+      = render partial: 'pulse_list_branches'
+      = render partial: 'pulse_list_comments'
+
+  .col
+    .card
+      .card-header
+        Requests
+      .card-body
+        - if @requests.any?
+          .row
+            .col
+              .progress
+                -# haml-lint:disable InlineStyles
+                - @requests_by_percentage.each do |state, percentage|
+                  %div{ title: "#{@requests_by_state[state]} #{state} requests",
+                        class: "progress-bar progress-state-#{state}",
+                        'aria-valuemax': '100', 'aria-valuemin': '0', 'aria-valuenow': percentage,
```

Comment:
[nitpick]

This could be written `aria: { valuemax: '100', valuemin: '0', valuenow: percentage }`